Snippets/FileShareDataBus/Core_8/CleanupStrategy/Define.cs

```diff
@@ -0,0 +1,17 @@
+namespace Core7.CleanupStrategy
```

Comment:
Should these namespaces change? Do these even really need namespaces?

Reply:
Pushed an update to remove all the unnecessary namespaces.